test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -322,6 +322,7 @@
     ("aten::_upsample_nearest_exact2d_backward", datetime.date(2022, 12, 15)),
     ("aten::_scaled_dot_product_attention", datetime.date(2023, 3, 15)),
     ("aten::_scaled_dot_product_flash_attention", datetime.date(2023, 3, 15)),
+    ("aten::_sparse_mask_helper", datetime.date(2023, 3, 15)),
     ("aten::_fused_sdp_choice", datetime.date(2023, 3, 15)),
```

Comment:
@cpuhrsch , @albanD , it is an OK to get it removed with the upcoming release, right?